addons/common/XEH_postInit.sqf

```diff
@@ -371,5 +371,23 @@ GVAR(deviceKeyCurrentIndex) = -1;
 {false},
 [0xC7, [true, false, false]], false] call cba_fnc_addKeybind;  //SHIFT + Home Key
 
+//Map opened/closed Events:
+GVAR(mapOpened) = false;
+[] spawn {
+    waitUntil {(!isNull findDisplay 12)};
```

Comment:
Is there no way to handle this with PFH wait or something? Not the first time I see it being used for map https://github.com/acemod/ACE3/blob/275b0c43000709282dafd5af89c949a12d8996be/addons/map/XEH_postInitClient.sqf#L15